components/brave_new_tab_ui/data/backgrounds.ts

```diff
@@ -4,83 +4,123 @@
 
 export const images: NewTab.Image[] = [
   {
-    'name': 'ntp-fall-2019-1',
-    'source': 'anders-jilden.webp',
-    'author': 'Anders Jildén',
-    'link': 'https://unsplash.com/@andersjilden?utm_source=unsplash&utm_medium=referral&utm_content=credit',
-    'originalUrl': 'https://unsplash.com/photos/uwbajDCODj4',
-    'license': 'https://unsplash.com/license'
+    'name': 'ntp-2020/2021-1',
+    'source': 'alex-plesovskich.avif',
+    'author': 'Alex Plesovskich',
+    'link': 'https://unsplash.com/@aples',
+    'originalUrl': 'Contributor sent the hi-res version through email',
+    'license': 'https://unsplash.com/photos/VPrTqd8B230'
   },
   {
-    'name': 'ntp-fall-2019-2',
-    'source': 'andreas-gucklhorn.webp',
-    'author': 'Andreas Gücklhorn',
-    'link': 'https://unsplash.com/@draufsicht?utm_source=unsplash&utm_medium=referral&utm_content=credit',
-    'originalUrl': 'https://unsplash.com/photos/mawU2PoJWfU',
-    'license': 'https://unsplash.com/license'
+    'name': 'ntp-2020/2021-2',
+    'source': 'andre-benz.avif',
+    'author': 'Andre Benz',
+    'link': 'https://unsplash.com/@trapnation',
+    'originalUrl': 'Contributor sent the hi-res version through email',
+    'license': 'https://unsplash.com/photos/axQXVkrUASg'
   },
   {
-    'name': 'ntp-fall-2019-3',
-    'source': 'andy-mai.webp',
-    'author': 'Andy Mai',
-    'link': 'https://unsplash.com/@veroz?utm_source=unsplash&utm_medium=referral&utm_content=credit',
-    'originalUrl': 'https://unsplash.com/photos/4knR_YzeUVc',
-    'license': 'https://unsplash.com/license'
+    'name': 'ntp-2020/2021-3',
+    'source': 'corwin-prescott_beach.avif',
+    'author': 'Corwin Prescott',
```

Comment:
Link is not specified for all Corwin Prescott images.
W/o link info, link will have NTP link.

Reply:
Due to missing link, license file generation is failed.
Can we get link for those images? @karenkliu @rebron (or explicitly put `chrome://newtab`?)
If we can't, license file generation script should be modified.

Reply:
Corwin doesn't have a link to provide. Maybe we should disable link? @rebron WDYT?

Reply:
Like below, `Corwin Prescott`'s credits is just a label.
![image](https://user-images.githubusercontent.com/6786187/99141760-cb6d8d80-2691-11eb-8d8e-157a2de273bb.png)

Reply:
Not this? `http://www.corwinprescott.com/`?